### ISSUE TITLE
Log level should be a parameter (and probably not default to debug)

### ISSUE THREAD

**Issue by TJM** [elastic/puppet-elasticsearch#1031]
* Module version: 6.3.3
* Puppet version: 5.5.8
* OS and version: CentOS 7.6

## Bug description

By default this module configures logging as debug, and other than providing the entire contents of log4j2.properties, it is not configurable. 

I think there should be a parameter for logging level, and it should probably be info (or whatever the package defaults) are.

Opinions?

~tommy

**Comment by tylerjl** [comment]
It's more than likely changed upstream over time, but I think that this module should default to whatever Elasticsearch ships as its default logging level, whatever that is.

**Comment by TJM** [comment]
Perhaps, I agree. The module should follow the default by default. Maybe debug is the default? either way I think it should be exposed as a parameter.

**Comment by batmac** [comment]
This is still a problem for us.
Should it be possible to make it a parameter to be able to customize it easily (as suggested by @TJM ) ?

**Comment by TJM** [comment]
Yes, it is quite easy, probably 2-3 lines of code, but seeing as how this module has over 20 PRs that have not seen much attention from "elastic" ... I am not sure how much good it will do to submit a fix. We are using a customized internal version with this "loglevel" and "ES7" fixes (at least).
